app/views/layouts/devise.html.erb

```diff
@@ -21,12 +67,31 @@
               <%= sanitize notice, :tags => %w(br) %>
             </div>
           <% end %>
-          <%= yield %>
-        </main>
+
+          <%# Actual login form %>
+          <div class="pt-4">
+            <%= yield %>
+          </div>
+        </div>
       </div>
+
+      <%# Announcements %>
+      <% announcements = get_announcements() %>
+      <% if announcements.present? %>
+        <div id="announcement-box" class="landing-box">
+          <i class="bi bi-megaphone p-1"></i>
+          <%= t('announcements')%>
+          <hr>
+          <%= get_announcements().html_safe %>
+        </div>
+      <% end %>
```

Comment:
I would personally prefer the announcement box above the login box, but maybe we should leave this discusion to basecamp.

Reply:
Ok, still moved it up for now as these announcements might be easily overlooked when they are at the bottom of the page. But of course, we can further discuss that on Basecamp ;)